diff --git a/aleph/logic/api_keys.py b/aleph/logic/api_keys.py
index original..modified 100644
--- a/aleph/logic/api_keys.py
+++ b/aleph/logic/api_keys.py
@@ -131,7 +131,7 @@ def hash_plaintext_api_keys():
             role.api_key_digest = hash_api_key(role.api_key)
             db.session.add(role)
             log.info(f"Hashing API key: {role}")
-        log.info(f"Comitting partition {index}")
+        log.info(f"Committing partition {index}")
         db.session.commit()

